docs/pages/dev-guide/developer-guide.md

```diff
@@ -38,6 +38,69 @@ Although all SDK services written today store metadata in Zookeeper, this is an
 
 They store the desired configuration of a service and all relevant information regarding Mesos tasks, respectively, but the precise format or location of the underlying data may be customized.  For example, the data may be stored in Zookeeper, but in a different format, or the data may be stored in a different persistent storage like etcd.  The defaults should be reasonable for most developers, however. Support for optional customization via drop-in replacement is a common pattern throughout the SDK.
 
+# Getting Started
+
+1. Create your framework
+
+   ```bash
+   $ ./new-service.sh frameworks/myframework
+   $ cd frameworks/myframework
+   ```
+
+   `new-service.sh` creates a skeleton framework.  You will extend
+   this skeleton.
+
+1. View `svc.yml`
+
+   Take a look at `src/main/dist/svc.yml`.  This is the YAML file that
+   defines your framework.  You will be editing this file.
+
+1. View `Main.java`
+
+   Take a look at
+   `src/main/java/com/mesosphere/sdk/mgummelt/scheduler/Main.java`.
+   This is the main method for your scheduler, which will be run in
+   DC/OS via Marathon.  It reads `svc.yml`, which defines its
+   behavior.  If you need any advanced functionality not provided by
+   YAML, such as complex deployment plans, you will write it here.
+
+1. Build a [package](#packaging)
+
+   ```bash
+   $ ./build.sh aws
+   ```
+
+   You will deploy your framework to DC/OS as a
+   [package](#packaging).  `build.sh` creates this package.
+
+1. Install your package.
+
+   `build.sh` should print instructions for installing the package.
+   They will look something like this:
+
+   ```bash
+   $ dcos package repo remove myframework-aws
+   $ dcos package repo add --index=0 myframework-aws https://mybucket.s3.amazonaws.com/stub-universe-myframework.zip
+   $ dcos package install --yes myframework
+   ```
+
+   You may now navigate to the DC/OS Services UI to view the
+   deployment.
+
+1. Uninstall your package.
+
+   ```
+   $ dcos package uninstall myframework
+   $ dcos node ssh --master-proxy --leader "docker run mesosphere/janitor /janitor.py -r myframework-role -p myframework-principal -z dcos-service-myframework
+   ```
+
+   The second command listed above is known as the **janitor**.  It is
```

Comment:
Suggest "The second command above runs the **janitor** script. The **janitor** script runs inside the DC/OS cluster to clean up Zookeeper state..."

Comma after "but, for now"